<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LidarConversionDialogBase</class>
 <widget class="QDialog" name="LidarConversionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lidar Conversion</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Input RGB file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QgsFileWidget" name="mQgsFileWidget"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Output Geotiff File Path</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QgsFileWidget" name="mQgsFileWidget_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
